c2corg_ui/static/js/user.js

```diff
@@ -96,6 +96,10 @@ app.UserController.prototype.logout = function() {
     this.alerts_.addSuccess('You have been disconnected');
   }.bind(this)).finally(function() {
     this.auth.removeUserData();
+    var path = this.ngeoLocation_.getPath();
+    if (path.indexOf('/edit/') !== -1 || path.indexOf('/account') !== -1) {
```

Comment:
Looks good but I wonder if a clearer syntax could be used (the kind of double negation makes the reading not that obvious). At least add a comment?